GX error, with Go version 1.11

Hi,
I'm the AUR ipget maintainer. When trying to fix the build, I got this error:
```
gx/ipfs/QmaPHkZLbQQbvcyavn8q1GFHg6o6yeceyHFSJ3Pjf3p3TQ/go-crypto/blake2b.supportAVX2: relocation target runtime.support_avx2 not defined
make: *** [Makefile:38: install] Error 2
```
This is all related to this issue: https://github.com/holochain/holochain-proto/issues/771
Hope you can fix this soon. 
Greetings,
Oskar

--- a/.gx/lastpubver
+++ b/.gx/lastpubver
@@ -1,1 +1,1 @@
-0.2.5: QmeCk9scee7q3X5hryzuT7HHhHYLVbKKHvnPpX1St2NWBe
+0.3.0: QmPjrHqppYZcDHqrjoikgzzchLJaVhd8aL9bhZdT8SfXgS

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,7 +5,7 @@ os:
 language: go
 
 go:
-    - 1.7
+    - 1.11
 
 script:
   - make build

--- a/Makefile
+++ b/Makefile
@@ -1,11 +1,11 @@
 # Minimum version numbers for software required to build IPFS
-IPFS_MIN_GO_VERSION = 1.7
-IPFS_MIN_GX_VERSION = 0.6
-IPFS_MIN_GX_GO_VERSION = 1.1
+IPFS_MIN_GO_VERSION = 1.11
+IPFS_MIN_GX_VERSION = 0.14
+IPFS_MIN_GX_GO_VERSION = 1.9
 
-dist_root=/ipfs/QmVQJ5mEGT38nYTfe3ZCCZPJhFDXTzBekUb8x3v8Pifugt
-gx_bin=bin/gx-v0.11.0
-gx-go_bin=bin/gx-go-v1.4.0
+dist_root=/ipfs/QmWrXF8Yj4bYymNnE8onjAQc6NJ5XtTa4N2GKB2Waz2qSQ
+gx_bin=bin/gx-v0.14.0
+gx-go_bin=bin/gx-go-v1.9.0
 
 # use things in our bin before any other system binaries
 export PATH := bin:$(PATH)

--- a/main.go
+++ b/main.go
@@ -7,9 +7,9 @@ import (
 	"strings"
 	"syscall"
 
-	path "gx/ipfs/QmQa2wf1sLFKkjHCVEbna8y5qhdMjL8vtTJSAc48vZGTer/go-ipfs/path"
-	cli "gx/ipfs/QmVcLF2CgjQb5BWmYFWsDfxDjbzBfcChfdHRedxeL3dV4K/cli"
-	fallback "gx/ipfs/Qmdk8Ea9GkbwHr7UNKVYaLRwwHSt69xBXuSvRVyNWZ9sZE/fallback-ipfs-shell"
+	path "gx/ipfs/QmT3rzed1ppXefourpmoZ7tyVQfsGPQZ1pHDngLmCvXxd3/go-path"
+	fallback "gx/ipfs/QmaWDhoQaV6cDyy6NSKFgPaUAGRtb4SMiLpaDYEsxP7X8P/fallback-ipfs-shell"
+	cli "gx/ipfs/Qmc1AtgBdoUHP8oYSqU81NRYdzohmF45t5XNwVMvhCxsBA/cli"
 )
 
 func main() {

--- a/package.json
+++ b/package.json
@@ -7,9 +7,21 @@
   "gxDependencies": [
     {
       "author": "Stephen Whitmore <noffle@ipfs.io>",
-      "hash": "Qmdk8Ea9GkbwHr7UNKVYaLRwwHSt69xBXuSvRVyNWZ9sZE",
+      "hash": "QmaWDhoQaV6cDyy6NSKFgPaUAGRtb4SMiLpaDYEsxP7X8P",
       "name": "fallback-ipfs-shell",
-      "version": "3.2.4"
+      "version": "3.2.6"
+    },
+    {
+      "author": "why",
+      "hash": "QmT3rzed1ppXefourpmoZ7tyVQfsGPQZ1pHDngLmCvXxd3",
+      "name": "go-path",
+      "version": "1.1.16"
+    },
+    {
+      "author": "urfave",
+      "hash": "Qmc1AtgBdoUHP8oYSqU81NRYdzohmF45t5XNwVMvhCxsBA",
+      "name": "cli",
+      "version": "1.19.1"
     }
   ],
   "gxVersion": "0.7.0",

--- a/package.json
+++ b/package.json
@@ -28,6 +28,6 @@
   "language": "go",
   "license": "",
   "name": "ipget",
-  "version": "0.2.5"
+  "version": "0.3.0"
 }
 

--- a/sharness/Makefile
+++ b/sharness/Makefile
@@ -40,12 +40,14 @@ $(SHARNESS): FORCE
 	lib/install-sharness.sh
 
 bin/ipget:
-	go build -o bin/ipget github.com/ipfs/ipget
+	mkdir -p bin
+	$(MAKE) -C .. build
+	cp ../ipget bin/ipget
 
 bin/ipfs:
 	mkdir -p bin
 	go get github.com/ipfs/ipfs-update
-	ipfs-update fetch v0.4.0 --output=bin/ipfs
+	ipfs-update fetch v0.4.18 --output=bin/ipfs
 
 bin/go-sleep:
 	go get -d github.com/chriscool/go-sleep

